www/wp-content/sunrise.php

```diff
@@ -148,11 +137,6 @@
 		wp_cache_add( $site_id, $current_site, 'wsuwp:network', 60 * 60 * 12 );
 	}
 
-	if ( isset( $current_blog->ssl_enabled ) && true === $current_blog->ssl_enabled && false === WSU_LOCAL_CONFIG ) {
-		define( 'FORCE_SSL_ADMIN', true );
```

Comment:
Going to show my ignorance here - should these constants still be set to true here, or is that taken care of elsewhere? Or no longer necessary at all?

Reply:
Good question! It would be best to also set `FORCE_SSL_ADMIN`, but there are a handful of cases where we move a site and the cert isn't ready yet.

Instead, the forcing of HTTPS is handled in the Nginx configuration. Every domain is forced to HTTPS as soon as we have a cert.

Reply:
Cool!